src/averageAges.js

```diff
@@ -15,12 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  let age = 0;
```

Comment:
```suggestion
  let age = (century === undefined) ? m.map(x => x.died - x.born)
    : a.map(x => x.died - x.born);
```
Для лаконічності можна не оголошувати тут змінну і присвоювати їй 0, а зразу ж нижче оголосити її і присвоїти значення